cpu/stm32l0/include/periph_cpu.h

```diff
@@ -81,6 +82,12 @@ typedef struct {
 #define EEPROM_SIZE                (2048U)  /* 2kB */
 #elif defined(CPU_LINE_STM32L031xx)
 #define EEPROM_SIZE                (1024U)  /* 1kB */
+#elif defined(CPU_LINE_STM32L010xB) || defined(CPU_LINE_STM32L011x3) || defined(CPU_LINE_STM32L011x4) || defined(CPU_LINE_STM32L011x4)
```

Comment:
`CPU_LINE_STM32L011x4` is listed twice, the `STM32L011x3` variant isn't handled anywhere else I think.

Reply:
Yes, it's a typo for `CPU_LINE_STM32L021x4`